docs/website/docs/developers/general/testing-guide.md

```diff
@@ -396,6 +396,8 @@ not supported by Bazel rules at this point.
 
 ## External test suites
```

Comment:
This page is published at https://iree.dev/developers/general/testing-guide/#external-test-suites. Generally trying to put enough information there so

* developers working in just this iree-org/iree repository can understand what the different tests are and how to handle newly failing or passing tests
* developers are aware of out of tree test suites
* each test suite is put in context

Along these lines, I would like to promote more of the test suite work going on (in both iree-test-suites and SHARK-TestSuite) up to the level of overall IREE ecosystem dashboards and release notes. For example, each stable release could highlight the test result delta and average performance delta since the previous release.